src/Data/Blob/IO.hs

```diff
@@ -50,6 +50,7 @@ readBlobsFromGitRepo path oid excludePaths = liftIO . fmap catMaybes $
       , lang `elem` codeNavLanguages
       , not (pathIsMinified path)
       , path `notElem` excludePaths
+      , null includePaths || path `elem` includePaths
```

Comment:
I have some mild discomfort around “what does it mean if we have the same path in both `excludePaths` and `includePaths`, and will this be surprising?” but since the CLI doesn’t allow you to specify both I think that’s ok 👍

Reply:
Ah yeah, if you call this function directly and pass the same path in both the excludes and includes list then excludes always win out. I think this might actually be fine/expected behavior.